mangaki/mangaki/models.py

```diff
@@ -141,6 +141,7 @@ def retrieve_poster(self, url=None, session=None):
             return False
 
         filename = os.path.basename(urlparse(url).path)
+        # Hé mais ça va pas écraser des posters / créer des collisions, ça ?
```

Comment:
Alright, then wouldn't that be better if you put the issue link in the comment so that anyone can access to discussion and context about the issue?
Also, I was merely exposing my thoughts about the problem :smile: